### PR TITLE
zh-trans: /docs/tasks/administer-cluster/limit-storage-consumption.md

### PR DESCRIPTION
**Pull request by zgfh**
zh-trans: /docs/tasks/administer-cluster/limit-storage-consumption.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+The admin would like to limit:
+-->
+管理员想要限制：
+
+<!--
+1. The number of persistent volume claims in a namespace
+2. The amount of storage each claim can request
+3. The amount of cumulative storage the namespace can have
+-->
+1. 命名空间中 persistent volume claims 的数量
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
1. 命名空间中持久卷申领（persistent volume claims）的数量
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+2. 每个 claim 可以请求的存储量
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
2. 每个申领（claim）可以请求的存储量
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+3. 命名空间可以具有的累积存储量
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
3. 命名空间可以具有的累计存储量
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+## LimitRange to limit requests for storage
+-->
+## LimitRange 限制存储请求
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
## 使用 LimitRange 限制存储请求
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+Adding a `LimitRange` to a namespace enforces storage request sizes to a minimum and maximum. Storage is requested via `PersistentVolumeClaim`. The admission controller that enforces limit ranges will reject any PVC that is above or below the values set by the admin.
+-->
+将 `LimitRange` 添加到命名空间会将存储请求大小强制设置为最小值和最大值。通过 `PersistentVolumeClaim` 请求存储。执行限制范围的许可控制器且拒绝任何高于或低于管理员设置的值的 PVC。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
将 `LimitRange` 添加到命名空间会为存储请求大小强制设置最小值和最大值。存储是通过 `PersistentVolumeClaim` 来发起请求的。执行限制范围控制的准入控制器会拒绝任何高于或低于管理员所设阈值的 PVC。
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+      storage: 2Gi
+    min:
+      storage: 1Gi
+```
+
+<!--
+Minimum storage requests are used when the underlying storage provider requires certain minimums. For example,
+AWS EBS volumes have a 1Gi minimum requirement.
+-->
+当底层存储提供程序需要某些最小值时，将使用最小存储请求。例如，AWS EBS volumes 的最低要求为 1Gi。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
当底层存储提供程序需要某些最小值时，将会用到所设置最小存储请求值。例如，AWS EBS volumes 的最低要求为 1Gi。
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+## StorageQuota to limit PVC count and cumulative storage capacity
+-->
+## StorageQuota 限制 PVC 数量和累积存储容量
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
## 使用 StorageQuota 限制 PVC 数目和累计存储容量
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+Admins can limit the number of PVCs in a namespace as well as the cumulative capacity of those PVCs. New PVCs that exceed
+either maximum value will be rejected.
+-->
+管理员可以限制命名空间中的 PVCs 数量以及这些 PVCs 的累积容量。超过任一最大值的新 PVCs 将被拒绝。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
管理员可以限制某个命名空间中的 PVCs 个数以及这些 PVCs 的累计容量。新 PVCs 请求如果超过任一上限值将被拒绝。
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+In this example, a 6th PVC in the namespace would be rejected because it exceeds the maximum count of 5. Alternatively,
+a 5Gi maximum quota when combined with the 2Gi max limit above, cannot have 3 PVCs where each has 2Gi. That would be 6Gi requested
+ for a namespace capped at 5Gi.
+-->
+在此示例中，命名空间中的第 6 个 PVC 将被拒绝，因为它超过了最大计数 5。或者，当与上面的 2Gi max 限制组合时，5Gi 最大配额不能具有 3 个 PVC，其中每个具有 2Gi。这将是 6Gi 要求的名称空间限制为 5Gi。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
在此示例中，命名空间中的第 6 个 PVC 将被拒绝，因为它超过了最大计数 5。或者，当与上面的 2Gi 最大容量限制结合在一起时，意味着 5Gi 的最大配额不能支持 3 个都是 2Gi 的 PVC。后者实际上是向命名空间请求 6Gi 容量，而该命令空间已经设置上限为 5Gi。
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+```
+
+{{% /capture %}}
+
+{{% capture discussion %}}
+
+<!--
+## Summary
+-->
+## 摘要
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
## 小结
```

File: content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+A limit range can put a ceiling on how much storage is requested while a resource quota can effectively cap the storage consumed by a namespace through claim counts and cumulative storage capacity. The allows a cluster-admin to plan their
+cluster's storage budget without risk of any one project going over their allotment.
+-->
+限制范围可以限制请求的存储量，而资源配额可以通过 claim 计数和累积存储容量有效地限制命名空间占用的存储。允许集群管理员规划其集群的存储预算，而不会有任何一个项目超过其分配的风险。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md]:
```suggestion
限制范围对象可以用来设置可请求的存储量上限，而资源配额对象则可以通过申领计数和累计存储容量有效地限制命名空间耗用的存储量。这两种机制使得集群管理员能够规划其集群存储预算而不会发生任一项目超量分配的风险。
```

**Comment by zgfh** [comment]
@tengqm 非常感谢，以更改

**Pull request merged**